gridcapa-core-valid-app/src/main/java/com/farao_community/farao/gridcapa_core_valid/app/RaoRunner.java

```diff
@@ -0,0 +1,20 @@
+package com.farao_community.farao.gridcapa_core_valid.app;
+
+import com.rte_france.farao.rao_runner.api.resource.RaoRequest;
+import com.rte_france.farao.rao_runner.api.resource.RaoResponse;
+import com.rte_france.farao.rao_runner.starter.RaoRunnerClient;
+import org.springframework.stereotype.Component;
+
+@Component
+public class RaoRunner {
```

Comment:
No need to create this class. It's better to use RaoRunnerClient from com.farao_community.farao.rao_runner.starter